src/worker/controller/actions/tasks/VerifyAndStoreResultAction.js

```diff
@@ -144,7 +144,7 @@ class VerifyAndStoreResultAction {
       if (result instanceof ComputeResult && !result.hasDelta()) {
         try {
           let tips = await this._controller.asyncExecCmd(constants.NODE_NOTIFICATIONS.GET_TIPS, {contractAddresses: contractAddress, useCache: false});
-          if (!tips || !tips[0].address || tips[0].address !== contractAddress) {
+          if (!tips || tips.length === 0 || !tips[0].address || tips[0].address !== contractAddress) {
```

Comment:
Also here. Better to do `!Array.isArray(tips)` instead of just `!tips`.  
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray